### PR TITLE
New package: git-secret-0.2.3

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/git-secret/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+short_desc="Bash-tool to store your private data inside a git repository"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="http://git-secret.io/"
+distfiles="https://github.com/sobolevn/git-secret/archive/v${version}.tar.gz"
+checksum=c821c25865ce7e13a67453debb6d60a8c1730102ecfc4c4b4c4858a02201ab26
+
+do_install() {
+	vbin git-secret
+	vmkdir usr/share/zsh/plugins/${pkgname}
```
Comment (maxice8) [inline srcpkgs/git-secret/template]: i think just vinstall is enough

**Comment by shizonic** [comment]
@maxice8 Removed.

**Comment by shizonic** [comment]
I think the build server has some problems installing some packages.
Please have a look at the log:
```
[*] Downloading binary packages
xbps-triggers-0.105_1.noarch.xbps: [8256B 49%] 7042KB/s ETA: 00m00s[K
xbps-triggers-0.105_1.noarch.xbps: 8256B [avg rate: 14MB/s][K
xbps-triggers-0.105_1.noarch.xbps.sig: [512B 100%] 20MB/s ETA: 00m00s[K
xbps-triggers-0.105_1.noarch.xbps.sig: 512B [avg rate: 20MB/s][K
base-files-0.139_10.x86_64-musl.xbps: [51KB 7%] 35KB/s ETA: 00m11s[K
base-files-0.139_10.x86_64-musl.xbps: 51KB [avg rate: 440KB/s][K
ERROR: [trans] failed to download `base-files-0.139_10' signature from `https://repo2.voidlinux.eu/current/musl': Operation timed out
[1m[31m=> ERROR: Failed to install bootstrap packages!
[m
travis_time:end:0b83da00:start=1533894614354577179,finish=1533894745830206633,duration=131475629454
[0K
[31;1mThe command "common/travis/bootstrap.sh $BOOTSTRAP" failed and exited with 1 during .[0m

Your build has been stopped.
```

**Comment by shizonic** [comment]
@maxice8 Thanks. Will do that asap.

File: srcpkgs/git-secret/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'git-secret'
+pkgname=git-secret
+version=0.2.3
+revision=1
```
Comment (maxice8) [inline srcpkgs/git-secret/template]: can add noarch=yes, there are no ELF binaries, /usr/bin/git-secret is a bash script

**Comment by maxice8** [comment]
please notify when you made requested changes otherwise i might miss while scanning

**Pull request merged**